src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/csv/CSVResultsExtractor.java

```diff
@@ -336,8 +336,16 @@ private String findFieldValue(String header, Map<String, Object> doc, boolean fl
 
     private String quoteValueIfRequired(final String input, final String separator) {
 
-        final String quote = input.contains(separator) ? "\"" : "";
-        return quote + input + quote + separator;
+        final String quote = "\"";
+        final String result;
```

Comment:
consider removing this variable and just returning values from functions in if, following the advice from

https://blog.codinghorror.com/flattening-arrow-code/
#4: Always opportunistically return as soon as possible from the function.

Reply:
I like the other advice which says minimize number of return statements in the function. It becomes really helpful when debugging the code, in other words putting 1 breakpoint instead of N. But this case is pretty simple, doesn't matter for me, will update.